components/checkbox.module.css

```diff
@@ -0,0 +1,22 @@
+.checkbox {
+  position: relative;
+  height: 24px;
+  width: 24px;
+  border: 1px solid #d1d6dc;
+  border-radius: 4px;
+
+  &:after {
+    content: "";
+    position: absolute;
+    top: -5px;
+    left: -5px;
+    height: calc(100% + 10px);
+    width: calc(100% + 10px);
+    border: 2px solid #dc2626;
+    border-radius: 8px;
+  }
+  &[aria-checked="true"] {
+    background: url(/images/checkmark.svg) no-repeat center;
```

Comment:
every rule in this file except the `background-image` on this line and the `calc`s can be added with `@apply windi-class-foo bar-baz` etc.

and if windi had a support for aria-checked we’d be able to do `checked:background-image-[url(/images/checkmark.svg/)]`

related, i would like to add iconify svg with auto-import but im also kind of holding out for windicss to just add them directly, as v4 will be adopting most of [unocss’s new goodies](https://antfu.me/posts/icons-in-pure-css) (anthony fu, who made this, is also on the windi team)